cmd/trello/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"github.com/merico-dev/stream/internal/pkg/plugin/trello"
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+// NAME is the name of this DevStream plugin.
+const NAME = "trello"
+
+// Plugin is the type used by DevStream core. It's a string.
+type Plugin string
+
+// Create implements the installation of trello board.
```

Comment:
"installation": creation might be better because Trello is a web service; we don't really "install" anything. Also, all usages of "Trello" should be the same as their official name (uppercase T).